src/heightmap.cpp

```diff
@@ -286,11 +286,18 @@ geo::ShapePtr createHeightMapShape(const cv::Mat& image_tmp, double resolution,
                 }
                 Q.pop();
             }
+            // Account for pixel thickness
+            p_min.x -= 1;
+            p_min.y -= 1;
```

Comment:
Conceptually its better to keep p_min as the coordinates of the pixel. Take thickness into account where needed.